.github/workflows/chatops.yml

```diff
@@ -293,11 +298,8 @@ jobs:
           make gotests/install
           make -j 4 gotests/gen
 
-          git config --global user.name "vdaas-ci"
-          git config --global user.email "ci@vdaas.org"
```

Comment:
Do I understand correctly that if this line is removed, the commit account will be automatically authenticated with the GPG key as a vdaas-ci?

Reply:
No.
ghaction-import-gpg sets them. There's no need to specify manually.